services/QuillLMS/app/lib/rule_feedback_history.rb

```diff
@@ -51,22 +51,29 @@ def self.feedback_history_to_json(f_h)
   end
 
   def self.generate_rulewise_report(rule_uid:, prompt_id:, start_date: nil, end_date: nil)
```

Comment:
Updated this code to make it safer. (Even though I think it's just staff running it, but we got a security issue)